packages/shared/src/node-apis/redis.ts

```diff
@@ -127,12 +152,36 @@ export default class RedisClient {
     return value ? broadcastParsers[chain].parse(JSON.parse(value)) : null;
   }
 
-  async getDeposits(chain: Chain, asset: Asset, address: string) {
+  async getDeposits(chain: Chain, asset: Asset, address: string): Promise<PendingDeposit[]> {
```

Comment:
now that this method returns the tx ref information, we should return it from the `status` endpoint. to do that, we need to adjust the `getPendingDeposit` method

Reply:
but would be fine for me to do that in a separate pr if you prefer that!

Reply:
Will do in separate pr 👾